dolphin-dnn/src/main/java/edu/snu/dolphin/dnn/layerparam/initializer/FullyConnectedLayerParameterInitializer.java

```diff
@@ -57,15 +60,11 @@ public FullyConnectedLayerParameterInitializer(
   /** {@inheritDoc} */
   @Override
   public LayerParameter generateInitialParameter() {
-    final INDArray weight = Nd4j.randn(numInput, numOutput, randomSeed);
-    final INDArray bias = Nd4j.valueArrayOf(1, numOutput, initBias);
+    final Matrix weight = matrixFactory.randn(numInput, numOutput, randomSeed);
+    final Matrix bias = matrixFactory.zeros(numOutput).fill(initBias);
```

Comment:
We should use `matrixFactory.create()` instead of `zeros()` to avoid zero-initialization cost.